packages/babel-traverse/src/path/introspection.js

```diff
@@ -402,10 +402,7 @@ export function _resolve(dangerous?, resolved?): ?NodePath {
 export function isConstantExpression() {
   if (this.isIdentifier()) {
     const binding = this.scope.getBinding(this.node.name);
-    if (!binding) {
-      return false;
-    }
-    return binding.constant && binding.path.get("init").isConstantExpression();
+    return binding && binding.constant;
```

Comment:
I'm personally not a fan of that, I would prefer something like:

```js
if (!binding) { return false }
return binding.constant;
```

Reply:
@nicolo-ribaudo the styling is wrong here, prettier will probably fix it for you?

Reply:
Actually prettier accepts it. I think that it is ok to have a single statement on the sam line of the if.